src/index.ts

```diff
@@ -46,8 +48,8 @@ export function map<T>(opt_initial: T | undefined) {
  * @returns {boolean}
  * @template T
  */
-export function hasOwn<T>(obj: T, key: string) {
-  return _hasOwn.call(obj, key);
+export function hasOwn<T>(object: T, key: string) {
```

Comment:
```suggestion
export const hasOwn = <T>(object: T, key: string): boolean {
```

I just noticed that the type for `key` is wrong. It should be a [`ESPropertyKey`](https://github.com/openinf/util-object/blob/a013a28d6beda5ccf222849f212c38525f1926ff/src/types.ts#L1), but we'll handle those in a separate PR since it's not relevant to this.